.gitignore

```diff
@@ -1,5 +1,7 @@
 pf.log
 _output
+# GoLand
```

Comment:
ok - but have you tried Visual Studio Code with the golang module?

Reply:
Yes I did. Worked fine.
But I was lucky to get subscription provided by Red Hat for GoLand and I do not think I am going to switch back to VS anytime soon :-)